std_msgs/Int16 obj_id
std_msgs/Bool pick_status
---

